HIP/hip-904.md

```diff
@@ -800,9 +895,9 @@ The record file emitted from a consensus node will contain
 - Alice intent to transfer 20 T7 tokens to Eric
 - Alice intent to transfer NFT serials 7 & 8 of token T8 to Eric
 
-Note: In the transaction body Alice had split her intended transfers to Dave in two but in the
-result exposed by the record they were combined. TokenAirdrops may be combined in an additive
-fashion. It is not possible to subtract. For this a sender may cancel and reissue their airdrop
+Note: TokenAirdrops for fungible tokens may be combined in an additive
+fashion. It is not possible to subtract. For this a sender may cancel and reissue their airdrop.
+Multiple TokenAirdrops for same NFT serial are not possible.
```

Comment:
The former text is better, as it uses an example.  This just makes an assertion that's covered elsewhere.

Reply:
I removed the reference to the example because it was not valid one (see https://github.com/hashgraph/hedera-improvement-proposal/pull/983#discussion_r1633442790).

Reply:
Note that we may have to include split amounts in two or more "airdrop" transactions in the examples to refer to that setup here.